File: website/client/components/language/swift/tachyon-wasm.swift
var count = 6
let label = "from Swift"

func doubled() -> Int { count * 2 }

let tac: [String: TacMember] = [
    "count": .field({ count }, { count = $0 as? Int ?? count }),
    "label": .field({ label }),
    "doubled": .method({ _ in doubled() }),
]

File: website/client/components/language/csharp/tachyon-wasm.cs
public static class Companion
{
    public static int Count = 6;
    public static string Label = "from C#";

    public static int Doubled() => Count * 2;

    public static readonly Dictionary<string, TacMember> Tac = new()
    {
        ["count"] = TacMember.Field(() => Count, value => Count = Convert.ToInt32(value)),
        ["label"] = TacMember.Field(() => Label),
        ["doubled"] = TacMember.Method(arguments => Doubled()),
    };
}
